client/hosting/overview/components/style.scss

```
@@ -208,6 +209,15 @@ $card-padding: 24px;
 	font-size: $font-body-extra-small;
 	line-height: 16px;
 	margin-top: 30px;
+
+	.plan-storage__bar .progress-bar__progress {
+		background-color: $blueberry-color;
```

```suggestion
		background-color: var(--theme-highlight-color);
```
This seems to achieve the same effect, and using a global variable for a theme-level color is preferable. 

@nightnei, I think I saw some PR where you had also defined a local `$blueberry-color` variable.

@chad1008, could you spin up a PR to replace those instances with `var(--theme-highlight-color)` (unless you see a good reason not to)?